verde/base/base_classes.py

```diff
@@ -353,40 +358,55 @@ def score(self, coordinates, data, weights=None):
 
     def grid(
         self,
+        coordinates=None,
```

Comment:
Put new arguments at the end so we don’t break code that doesn’t use keyword arguments. We should set keyword-only arguments from 2.0.0 to avoid this.

Reply:
You're right! Moving them to the bottom.